sys/linux/dev_video4linux_vim2m.txt

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# V4L2 specific support for the vim2m driver.
+#
+# These descriptions narrow the search space to focus on the M2M
+# framework code. The vim2m driver should be built into the kernel
+# (CONFIG_VIDEO_VI2M2=y)
+
+include <linux/time.h>
+include <linux/types.h>
+include <uapi/asm/ioctl.h>
+include <uapi/linux/fcntl.h>
+include <uapi/linux/videodev2.h>
+include <uapi/linux/v4l2-common.h>
+include <uapi/linux/v4l2-subdev.h>
+include <uapi/linux/v4l2-mediabus.h>
+include <uapi/linux/media-bus-format.h>
+include <uapi/linux/v4l2-controls.h>
+
+resource fd_vim2m[fd]
+
+# syz_open_dev will use devices from /dev/video0 to /dev/video10
+# openat$vim2m assumes a symlink (/dev/vim2m) to the appropriate vim2m device
+# This can be set with a udev rule such as this:
+#
+#     ATTR{name}=="vim2m", SYMLINK+="vim2m"
```

Comment:
This needs to be appended to some file in the image, right?
Please add it somewhere here:
https://github.com/google/syzkaller/blob/master/tools/create-gce-image.sh#L140
(you will also need to run 'make generate' to update some generated files after this).
This will enable it on syzbot and also serve as some scripted reference to other users.

Reply:
Yes, it's nicer than alternatives I could think of. I wasn't aware of udev.
Yes, tools/create-image.sh would be nice to update as well (but it's not used by syzbot).